cypress/integration/login.js

```diff
@@ -24,4 +24,12 @@ describe('Sourcebooks login', function() {
         cy.get('.user-info__title').contains('Demo User');
         cy.get('.main-nav').find('li').should('have.length', 1);
     })
+
+    //test
+    it('Should validate todays date', function () {
+
+        var today = new Date();
+

```

Comment:
I would personally delete this line for cleaner spacing, this is a minor thing :)